Desktop/Prework/Allegro/allegro/src/App.module.scss

```diff
@@ -0,0 +1,31 @@
+.mainWrapper{
+    margin: 4vh 14vw 0 14vw;
+    display: grid;
+    grid-gap: 0;
+    // grid-gap: 1rem;
+    grid-template-columns: repeat(4, 1fr);
+    grid-auto-rows: 19rem;
+    // grid-column-gap: 7rem;
+}
+.footer{
+    display: flex;
+    justify-content: center;
+    height: 5rem;
+}
+.container{
+    background-color: black;
+}
+
+@media only screen and (max-width: 600px) {
```

Comment:
What about printing?

I am just asking, because below we use `@media (min-width: 600px) and (max-width: 1024px)`, which doesn't have `only screen` constraint. It would be good to be consistent.